### DISCUSSION TITLE
Bacheca studenti/docenti

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#262] [Idee e proposte]
Salve, sarebbe possibile integrare una sezione con scritto bacheca per inserire materiale e far mandare dagli studenti i compiti come in argo/axios/classeviva?

**Comment by trinko** [comment]
No, non è previsto. 
A mio avviso questa è una funzionalità per una piattaforma didattica (Google Workspace, Moodle, ecc.), non per un registro elettronico. Tra l'altro, le piattaforme didattiche hanno funzioni molto più avanzate di questa.
